tst/standard/display.tst

```diff
@@ -201,6 +201,12 @@ node [shape=Mrecord, height=0.5, fixedsize=true]ranksep=1;
 2 -> 1
 }
 
+# Splash
+gap> Splash(DotDigraph(RandomDigraph(10)), rec(viewer := 1));
+Error, the option `viewer` must be a string, not an integer,
+gap> Splash(DotDigraph(RandomDigraph(10)), rec(viewer := "shit"));
```

Comment:
interesting

Reply:
Also, there is a fairly popular program called 'fuck' ( https://github.com/nvbn/thefuck ) which I believe installs a command called 'shit', so some people will have that program installed (if you are going to look for a runnable program with that name).

Reply:
So, should I use a string that can't be misconstrued as a program?? Or is this good to go? @wilfwilson

Reply:
I'd prefer that (don't mind if you still want to have 'shit' as part of it), but I don't feel strongly about it.

Reply:
@james-d-mitchell Okay fine, but maybe write "shite" instead to avoid the conflict potential that Chris mentioned above?
```suggestion
gap> Splash(DotDigraph(RandomDigraph(10)), rec(viewer := "shite"));
```